File: applicatie/src/main/java/com/vdzon/administratie/checkandfix/rest/CheckAndFixResource.java
package com.vdzon.administratie.checkandfix.rest;

import com.vdzon.administratie.util.JsonUtil;
import spark.Spark;

import javax.inject.Inject;


public class CheckAndFixResource {

    @Inject
    public CheckAndFixResource(CheckAndFixService rubricecheckAndFixServicerService) {
        Spark.get("/rest/checkandfixregels/", rubricecheckAndFixServicerService::getCheckAndFixRegels, JsonUtil.json());
        Spark.post("/rest/fix/", rubricecheckAndFixServicerService::fix, JsonUtil.json());
    }


}

File: rest/src/main/java/com/vdzon/administratie/rest/checkandfix/CheckAndFixResource.kt
package com.vdzon.administratie.rest.checkandfix

import com.vdzon.administratie.util.JsonUtil
import spark.Route
import spark.Spark

import javax.inject.Inject


class CheckAndFixResource
@Inject
constructor(rubricecheckAndFixServicerService: CheckAndFixService) {
    init {
        Spark.get("/rest/checkandfixregels/", Route { req, res -> rubricecheckAndFixServicerService.getCheckAndFixRegels(req, res) }, JsonUtil.json())
        Spark.post("/rest/fix/", Route { req, res -> rubricecheckAndFixServicerService.fix(req, res) }, JsonUtil.json())
    }
}
